docker-compose.yml

```diff
@@ -220,6 +220,20 @@ services:
     depends_on:
       - solr1
 
+
+  embeddings:
```

Comment:
David mentioned that embeddings service seems to also use a lot of memory potentially. It might be worth implementing https://docs.docker.com/compose/compose-file/compose-file-v3/#resources